rca/static_src/javascript/programmes/components/ProgrammesResults/ProgrammesResults.scss

```diff
@@ -0,0 +1,111 @@
+.programmes-results {
+    $root: &;
+    padding-top: 0;
+
+    @include media-query(large) {
+        padding-top: $gutter * 2;
+    }
+
+    &__count {
+        grid-column: 1 / span 2;
+        margin-bottom: $gutter * 1.5;
+
+        @include media-query(large) {
+            grid-column: 2 / span 2;
+        }
+
+        .bg--dark & {
+            color: $color--meta-darkbg;
+        }
+
+        .bg--light & {
+            color: $color--meta-lightbg;
+        }
+    }
+
+    &__actions {
+        grid-column: 1 / span 1;
+        margin-bottom: $gutter * 0.5;
+
+        @include media-query(medium) {
+            margin-bottom: $gutter * 0.5;
```

Comment:
can this go if it's the same as the style for mobile?

Reply:
Nice catch!